docker-compose.e2e.yml

```diff
@@ -0,0 +1,43 @@
+version: '3.7'
+
+services:
+  e2e_pgdb:
+    container_name: e2e_pgdb
+    image: postgres:14-alpine
+    restart: unless-stopped
+    environment:
+      - POSTGRES_DB=${DB_NAME}
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+    expose:
+      - 5432:5432
+    volumes:
+      - ./.volumes/pgdata-${CHAIN_ID}/:/var/lib/postgresql/data
+
+  e2e_keys_api:
+    container_name: e2e_keys_api
+    build:
+      context: ./
+      dockerfile: Dockerfile.e2e
+    environment:
+      - NODE_ENV=production
+      - PORT=${PORT}
+      - CORS_WHITELIST_REGEXP=${CORS_WHITELIST_REGEXP}
```

Comment:
maybe we should leave here only required envs

Reply:
These vars only for local running.

Reply:
removed.